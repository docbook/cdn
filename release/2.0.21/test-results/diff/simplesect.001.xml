<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Chapter 1. Chapter wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="chapter">
         <header deltaxml:deltaV2="A=B" class="chapter-titlepage">
            <h2>Chapter 1. Chapter wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#R.1.3">1. A Section Title</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#R.1.4">2. Another Section Title</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#R.1.4.3">2.1. A Subsection Title</a>
                           </span>
                        </li>
                     </ul>
                  </li>
               </ul>
            </div>
         </div>
         <p deltaxml:deltaV2="A">This test case is for simplesect.</p><div deltaxml:deltaV2="B" class="content">
            <p>This test case is for simplesect.</p>
            <section id="R.1.3" class="section">
               <div class="section-titlepage">
                  <h2>1. A Section Title</h2>
               </div>
               <div class="content">
                  <p>Some content <span class="acronym">here</span>.</p>
                  <section id="R.1.3.3" class="simplesect">
                     <div class="section-titlepage">
                        <h3>First Simplesect</h3>
                     </div>
                     <div class="content">
                        <p>Some content</p>
                     </div>
                  </section>
               </div>
            </section>
            <section id="R.1.4" class="section">
               <div class="section-titlepage">
                  <h2>2. Another Section Title</h2>
               </div>
               <div class="content">
                  <p>Some content</p>
                  <section id="R.1.4.3" class="section">
                     <div class="section-titlepage">
                        <h3>2.1. A Subsection Title</h3>
                     </div>
                     <div class="content">
                        <p>Some content</p>
                        <section id="R.1.4.3.3" class="simplesect">
                           <div class="section-titlepage">
                              <h4>Second Simplesect</h4>
                              <h5>Subtitle</h5>
                           </div>
                           <div class="content">
                              <p>Some content</p>
                           </div>
                        </section>
                     </div>
                  </section>
               </div>
            </section>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="R.1.3" class="section">
            <div class="section-titlepage">
               <h2>1. A Section Title</h2>
            </div>
            <p>Some content <span class="acronym">here</span>.</p>
            <section id="R.1.3.3" class="simplesect">
               <div class="section-titlepage">
                  <h3>First Simplesect</h3>
               </div>
               <p>Some content</p>
            </section>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="R.1.4" class="section">
            <div class="section-titlepage">
               <h2>2. Another Section Title</h2>
            </div>
            <p>Some content</p>
            <section id="R.1.4.3" class="section">
               <div class="section-titlepage">
                  <h3>2.1. A Subsection Title</h3>
               </div>
               <p>Some content</p>
               <section id="R.1.4.3.3" class="simplesect">
                  <div class="section-titlepage">
                     <h4>Second Simplesect</h4>
                     <h5>Subtitle</h5>
                  </div>
                  <p>Some content</p>
               </section>
            </section>
         </section>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>