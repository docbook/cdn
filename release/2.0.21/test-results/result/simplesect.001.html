<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for simplesect.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: simplesect.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">5</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Chapter 1. Chapter wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="chapter"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Chapter 1. Chapter wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.3"</span>&gt;1. A Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.4"</span>&gt;2. Another Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                        <span class="tag">&lt;li&gt;
                           <span class="tag">&lt;span&gt;
                              <span class="tag">&lt;a <span class="att">href="#R.1.4.3"</span>&gt;2.1. A Subsection Title&lt;/a&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/li&gt;</span>
                     &lt;/ul&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff del">&lt;p&gt;This test case is for simplesect.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.3"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;1. A Section Title&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;Some content <span class="tag">&lt;span <span class="att">class="acronym"</span>&gt;here&lt;/span&gt;</span>.&lt;/p&gt;</span>
            <span class="tag">&lt;section <span class="att">class="simplesect"</span> <span class="att">id="R.1.3.3"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;First Simplesect&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
            &lt;/section&gt;</span>
         &lt;/section&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.4"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;2. Another Section Title&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
            <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.4.3"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;2.1. A Subsection Title&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
               <span class="tag">&lt;section <span class="att">class="simplesect"</span> <span class="att">id="R.1.4.3.3"</span>&gt;
                  <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                     <span class="tag">&lt;h4&gt;Second Simplesect&lt;/h4&gt;</span>
                     <span class="tag">&lt;h5&gt;Subtitle&lt;/h5&gt;</span>
                  &lt;/div&gt;</span>
                  <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
               &lt;/section&gt;</span>
            &lt;/section&gt;</span>
         &lt;/section&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Chapter 1. Chapter wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="chapter"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="chapter-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Chapter 1. Chapter wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.3"</span>&gt;1. A Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#R.1.4"</span>&gt;2. Another Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                        <span class="tag">&lt;li&gt;
                           <span class="tag">&lt;span&gt;
                              <span class="tag">&lt;a <span class="att">href="#R.1.4.3"</span>&gt;2.1. A Subsection Title&lt;/a&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/li&gt;</span>
                     &lt;/ul&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;This test case is for simplesect.&lt;/p&gt;</span>
            <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.3"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;1. A Section Title&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;Some content <span class="tag">&lt;span <span class="att">class="acronym"</span>&gt;here&lt;/span&gt;</span>.&lt;/p&gt;</span>
                  <span class="tag">&lt;section <span class="att">class="simplesect"</span> <span class="att">id="R.1.3.3"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;First Simplesect&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/section&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
            <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.4"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;2. Another Section Title&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
                  <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="R.1.4.3"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;2.1. A Subsection Title&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
                        <span class="tag">&lt;section <span class="att">class="simplesect"</span> <span class="att">id="R.1.4.3.3"</span>&gt;
                           <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                              <span class="tag">&lt;h4&gt;Second Simplesect&lt;/h4&gt;</span>
                              <span class="tag">&lt;h5&gt;Subtitle&lt;/h5&gt;</span>
                           &lt;/div&gt;</span>
                           <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                              <span class="tag">&lt;p&gt;Some content&lt;/p&gt;</span>
                           &lt;/div&gt;</span>
                        &lt;/section&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/section&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3">1. A Section Title</a></span></li><li><span><a href="#R.1.4">2. Another Section Title</a></span><ul class="toc"><li><span><a href="#R.1.4.3">2.1. A Subsection Title</a></span></li></ul></li></ul></div></div><p>This test case is for simplesect.</p><section id="R.1.3" class="section"><div class="section-titlepage"><h2>1. A Section Title</h2></div><p>Some content <span class="acronym">here</span>.</p><section id="R.1.3.3" class="simplesect"><div class="section-titlepage"><h3>First Simplesect</h3></div><p>Some content</p></section></section><section id="R.1.4" class="section"><div class="section-titlepage"><h2>2. Another Section Title</h2></div><p>Some content</p><section id="R.1.4.3" class="section"><div class="section-titlepage"><h3>2.1. A Subsection Title</h3></div><p>Some content</p><section id="R.1.4.3.3" class="simplesect"><div class="section-titlepage"><h4>Second Simplesect</h4><h5>Subtitle</h5></div><p>Some content</p></section></section></section></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3">1. A Section Title</a></span></li><li><span><a href="#R.1.4">2. Another Section Title</a></span><ul class="toc"><li><span><a href="#R.1.4.3">2.1. A Subsection Title</a></span></li></ul></li></ul></div></div><div class="content"><p>This test case is for simplesect.</p><section id="R.1.3" class="section"><div class="section-titlepage"><h2>1. A Section Title</h2></div><div class="content"><p>Some content <span class="acronym">here</span>.</p><section id="R.1.3.3" class="simplesect"><div class="section-titlepage"><h3>First Simplesect</h3></div><div class="content"><p>Some content</p></div></section></div></section><section id="R.1.4" class="section"><div class="section-titlepage"><h2>2. Another Section Title</h2></div><div class="content"><p>Some content</p><section id="R.1.4.3" class="section"><div class="section-titlepage"><h3>2.1. A Subsection Title</h3></div><div class="content"><p>Some content</p><section id="R.1.4.3.3" class="simplesect"><div class="section-titlepage"><h4>Second Simplesect</h4><h5>Subtitle</h5></div><div class="content"><p>Some content</p></div></section></div></section></div></section></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/simplesect.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>