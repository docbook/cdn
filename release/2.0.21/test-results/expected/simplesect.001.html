<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Chapter 1. Chapter wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. Chapter wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3">1. A Section Title</a></span></li><li><span><a href="#R.1.4">2. Another Section Title</a></span><ul class="toc"><li><span><a href="#R.1.4.3">2.1. A Subsection Title</a></span></li></ul></li></ul></div></div><p>This test case is for simplesect.</p><section id="R.1.3" class="section"><div class="section-titlepage"><h2>1. A Section Title</h2></div><p>Some content <span class="acronym">here</span>.</p><section id="R.1.3.3" class="simplesect"><div class="section-titlepage"><h3>First Simplesect</h3></div><p>Some content</p></section></section><section id="R.1.4" class="section"><div class="section-titlepage"><h2>2. Another Section Title</h2></div><p>Some content</p><section id="R.1.4.3" class="section"><div class="section-titlepage"><h3>2.1. A Subsection Title</h3></div><p>Some content</p><section id="R.1.4.3.3" class="simplesect"><div class="section-titlepage"><h4>Second Simplesect</h4><h5>Subtitle</h5></div><p>Some content</p></section></section></section></article><script src="../../resources/js/prism.js"></script></body></html>